moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useProjectListViewQuery.js

```diff
@@ -39,7 +39,18 @@ export const useGetProjectListView = ({
       ${queryString}
     `;
 
-    const exportQueryString = gql`{
+    return { query };
+  }, [
+    queryLimit,
+    queryOffset,
+    columnsToReturn,
+    orderByColumn,
+    orderByDirection,
+    searchWhereString,
+    advancedSearchWhereString,
+  ]);
+  const { exportQuery } = useMemo(() => {
+    const exportQueryString = gql`query ProjectListExport {
```

Comment:
separating this single useMemo into two useMemos so we dont have the `ProjectListView` query recalculating when the `ProjectListExport` inputs change.

Reply:
nice—that's definitely what this hook needs. since the queries are identical except for the limit and offset, would you be up for taking this a step further and making this hook just return a single query, that includes the limit and offset if they are provided?

then in the ProjectListViewTable, you can call the `useGetProjectListView()` hook twice, once for the list query and once for the export query. 

i also don't mind moving that task into a separate issue—i'd like to get that hook DRY'd out.